BLSQ/iaso
# IA-3058: Feature flag not "scrollable" enough

Explain what problem this PR is resolving
- Feature flag not "scrollable" enough
Related JIRA tickets : [IA-3058](https://bluesquare.atlassian.net/browse/IA-3058), [IA-2959](https://bluesquare.atlassian.net/browse/IA-2959)

## Self proofreading checklist


## Changes
- Use ConfirmCancelModal
- Change the maxWidth of the modal when on featureFlag is selected
- Reduce the aucomplete listbox height

## How to test
-Create or Edit a project add more featureFlags to it
- Check if in the dropdown list all featureFlags are available


## Print screen / video
[Screencast from 2024-06-25 08-46-26.webm](https://github.com/BLSQ/iaso/assets/19631540/52e8d474-4672-4aae-814b-b10fef2d0bee)


[IA-3058]: https://bluesquare.atlassian.net/browse/IA-3058?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[IA-2959]: https://bluesquare.atlassian.net/browse/IA-2959?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
